docs/specification/appendix.md

```diff
@@ -107,6 +107,28 @@ The IDL definition of the type specified by `name` along with all dependent type
 - a line containing exactly 80 `=` characters, then
 - A line containing only `IDL: <package resource name>` for that definition. The space between `IDL:` and the package resource name is mandatory. The package resource name does not include a file extension.
 
+### omgidl
+
+The `omgidl` schema encoding stores [OMG IDL](https://www.omg.org/spec/IDL/4.2/About-IDL) definitions for (X)CDR-encoded messages.
+
+- `name`: The globally-scoped name of the message type, eg. `my_company::my_module::MyType`
+- `encoding`: `omgidl`
+- `data`: valid OMG IDL definition file contents with a definition for the message type and all referenced types.
+
+#### Producing `schema.data` for the `omgidl` schema encoding
```

Comment:
Would using the exact same format as `ros2idl` not work? Why do these differ?

Reply:
people not using ROS 2 don't have package resource names for their types, which the ros2idl spec depends on.

Reply:
I'd also add that if we can avoid the "ros2idl" format is required for ROS2 messages since there's no native way to have multiple messages in a single "schema". In IDL, to our knowledge, it is valid to have multiple modules and structs in a single file - so the approach proposed in this PR results in a schema file that is still a valid IDL schema file and requires no additional "separator" scheme.